### PR TITLE
#586 properly initialize frozen property of VRRemote and SVCs + renaming

### PR DESCRIPTION
**Pull request by rosiereflo**
closes #568

**Checklist before requesting a review**

*use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes*

- [ ] unit tests and non regression tests were added (update of inputs, outputs, SA, algos)
- [ ] main documentation was updated (update of input/output file)
- [ ] if this commit targets a release branch: the corresponding milestone was added in the ticket and in this PR

### CODE REVIEW

File: sources/Outputs/src/ParDynModel.cpp [hunk 1]
```diff
@@ -108,13 +109,19 @@ ParDynModel::writeSVCParameterSet(const inputs::SettingDataBase::Set& set, const
       if (genDefinition.hasTransformer())
         new_set->addParameter(helper::buildParameter("secondaryVoltageControl_XTfoPu_" + std::to_string(idx) + "_",
                                                      (genDefinition.isNetwork()) ? constants::generatorNucXPuValue : constants::generatorXPuValue));
+
+      if (genDefinition.q < genDefinition.qmax && genDefinition.q > genDefinition.qmin) {
```
Comment (marcochiaramello) [inline sources/Outputs/src/ParDynModel.cpp:113]: I'd align the codition as in the modelica model (check on volatge values also and <= >=)

File: sources/Algo/include/GeneratorDefinitionAlgorithm.h [hunk 2]
```diff
@@ -196,13 +199,13 @@ class GeneratorDefinitionAlgorithm {
    * @brief Constructor
    *
    * @param gens generators list to update
-   * @param busesWithDynamicModel map of bus ids to a generator that regulates them
+   * @param busesRegulatedBySeveralGenerators map of bus ids to a generator that regulates them
```
Comment (marcochiaramello) [inline sources/Algo/include/GeneratorDefinitionAlgorithm.h:202]: thanks for renaming this!

**Review by marcochiaramello — CHANGES_REQUESTED** [review]

File: sources/Outputs/src/ParDynModel.cpp (hunk 1 above)
Comment (rosiereflo) [reply sources/Outputs/src/ParDynModel.cpp:113]: Regarding the voltages, it is not possible as initially URef=URegulated=targetV of the node

Comment (marcochiaramello) [reply sources/Outputs/src/ParDynModel.cpp:113]: I see, ok for me then! thx! we can still have <= and >=, I think

**Review by marcochiaramello — CHANGES_REQUESTED** [review]

File: sources/Outputs/src/ParDynModel.cpp (hunk 1 above)
Comment (marcochiaramello) [reply sources/Outputs/src/ParDynModel.cpp:113]: never mind, it is correct as it is

**Review by marcochiaramello — APPROVED** [review]

**Pull request merged**